host/CMakeLists.txt

```diff
@@ -1,6 +1,16 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+if (UNIX)
+# Generate OE repo config file
+set(REPO_CONFIG_FILE ../../host/repo_config_info.h)
+add_custom_target(generate_repo_config
+    COMMAND rm -f ${REPO_CONFIG_FILE}
```

Comment:
@andschwa Can you recommend a better way of doing this? 

I would certainly recommend avoiding changes to the source tree at build time at least. It seems to me that instead of trying to use `#define`, we should be able to just specify compiler variables directly in the CMake environment with the `-DOE_REPO_BRANCH_NAME=…` which would also remove the need for a separate shell script.

Reply:
This is a great suggestion.
I updated the CMakeLists.txt and removed this new script file that you pointed out. It's much cleaner now.